digiwf-cypress/cypress/pages/page.js

```diff
@@ -0,0 +1,54 @@
+class Page{
+
+    checkStartConditions(){
+        cy.log("Fails if there are any open tasks")
+        cy.get('.v-list-item--active > div:nth-child(1) > div:nth-child(1) > span:nth-child(2)').should('not.exist');
+    }
```

Comment:
Das erscheint mir hier nicht richtig, Nicht jede Page hat eine Liste (von offenen Tasks)

Reply:
Mir geht es nicht darum, dass der Check falsch ist, sondern das er hier doch in der Page für alle Views gilt, was aber nicht bei jeder View der Fall ist. Vielleicht reden wir aneinander vorbei😅

Reply:
> Hier schaue ich in der seitenleiste nach, ob hinter meine Aufgaben aktuell ein Wert steht.

ahhhhhhhhh.......sorry, dann habe ich den check falsch verstanden.

Vielleicht dann einfach einen besseren Namen wählen, da nicht klar ist, was die startCondition ist